app/views/provider_interface/decisions/new_offer.html.erb

```diff
@@ -19,10 +19,13 @@
 
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <%= f.govuk_check_boxes_fieldset :standard_conditions, legend: { text: 'Standard conditions (optional)', size: 'm', tag: 'h2' } do %>
-          <%= f.govuk_check_box :standard_conditions, 'Fitness to Teach check', label: { text: 'Fitness to Teach check' } %>
-          <%= f.govuk_check_box :standard_conditions, 'Disclosure and Barring Service (DBS) check', label: { text: 'Disclosure and Barring Service (DBS) check' } %>
-      <% end %>
+      <%= f.govuk_collection_check_boxes(
+            :standard_conditions,
+            standard_conditions_checkboxes,
+            :id,
+            :name,
+            legend: { text: 'Standard conditions (optional)', size: 'm', tag: 'h2' },
```

Comment:
should this say 'optional'?

Reply:
Hmm I haven't changed this text from the original copy. They are optional in that there is no validation to ensure these items are selected but I wonder about the wording...

Reply:
@mnacos https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1480/commits/d1d0ce58bcb7eb33c58ff82c9735268f3b68c24d

Reply:
Thanks, I think it's good to merge.